js/examples/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "irma-backend-examples",
-  "version": "0.1.0",
+  "version": "0.1.1",
```

Comment:
We don't release the examples, so don't necessarily have to update the version here.